requirements.txt

```diff
@@ -99,10 +101,15 @@ Sphinx~=5.0.2
 sphinx-rtd-theme~=1.0.0
 sphinxcontrib-applehelp~=1.0.2
 sphinxcontrib-devhelp~=1.0.2
+sphinx-favicon~=0.2
 sphinxcontrib-htmlhelp~=2.0.0
 sphinxcontrib-jsmath~=1.0.1
 sphinxcontrib-qthelp~=1.0.3
 sphinxcontrib-serializinghtml~=1.1.5
+sphinxcontrib-towncrier~=0.2.1a0
+sphinx-jsonschema~=1.19.1
+sphinxcontrib-apidoc~=0.3.0
+sphinx-issues~=3.0.1
```

Comment:
Are you intending to setup sphinx to use these things in this pr

Reply:
Let's not in this PR and we can add them in as needed in follow up pr's. I removed the none needed dep.